lambdas/postgres-migration-async-operation/src/index.ts

```diff
@@ -1,28 +1,31 @@
 'use strict';
 
+import Logger from '@cumulus/logger';
+import { DataMigration2HandlerEvent } from '@cumulus/types/migration';
+
 const asyncOperations = require('@cumulus/async-operations');
-const Logger = require('@cumulus/logger');
 const { models } = require('@cumulus/api');
 
 const logger = new Logger({ sender: '@cumulus/postgres-migration-async-operation' });
-exports.handler = async () => {
+
+exports.handler = async (event: DataMigration2HandlerEvent) => {
   const dynamoTableName = process.env.AsyncOperationsTable;
 
-  logger.info('Starting Postgres Migration Async Operation');
-  const asyncOperation = await asyncOperations.startAsyncOperation({
+  logger.info('About to start async operation for Data Migration 2');
+  const asyncOperationId = await asyncOperations.startAsyncOperation({
     cluster: process.env.EcsCluster,
     lambdaName: process.env.MigrationLambda,
     asyncOperationTaskDefinition: process.env.AsyncOperationTaskDefinition,
     description: 'Data Migration 2 Lambda ECS Run',
     operationType: 'Data Migration',
-    payload: {},
+    payload: event,
     stackName: process.env.stackName,
     systemBucket: process.env.system_bucket,
     dynamoTableName,
     knexConfig: process.env,
     useLambdaEnvironmentVariables: true,
   }, models.AsyncOperation);
 
-  logger.info('Completed Postgres Migration Async Operation');
-  return asyncOperation;
+  logger.info(`Started async operation ${asyncOperationId} for Data Migration 2`);
```

Comment:
The previous log message wasn't really accurate. The async operation has not completed when this line is reached, it has just been started and an ID has been returned. The operation completes asynchronously to the process that starts it

Reply:
That's a good point. Thanks for updating this to be more accurate!